grid2op/tests/test_ma.py

```diff
@@ -1005,6 +1031,26 @@ def test_action_spaces(self):
             # check name of classes are correct
             assert re.sub("^SubGridAction", "", type(do_nothing).__name__) == re.sub("^SubGridActionSpace", "", type(ma_env.action_spaces[agent]).__name__)
 
+    
+    def test_step(self):
+        
+        np.random.seed(0)
+        # TODO seed
+        self.ma_env.reset()
+        for _ in range(10):
+            while True:
+                actions = dict(
+                    zip(
+                        self.ma_env.agents, 
+                        [self.ma_env.action_spaces[agent].sample() for agent in self.ma_env.agents]
+                    )
+                )
+                obs, rewards, dones, info = self.ma_env.step(actions)
```

Comment:
And ?

You don't test the outcome ? Compare to known values etc? 

This test is like a "assert True"... Not really usefull

Reply:
It "tests" if any errors will be raised in the process.